### ISSUE TITLE
[FEATURE] Kindly add the Support for the updated file picker version 8.0.0

### ISSUE THREAD

**Issue by jaypathak9460** [the-airbender/quill_html_editor#184]
Because quill_html_editor >=2.2.8 depends on file_picker ^6.1.1 and myProject depends on file_picker ^8.0.0+1, quill_html_editor >=2.2.8 is forbidden.
So, because myProject depends on quill_html_editor ^2.2.8, version solving failed.

here i want to upgrade both the packages but quill_html_editor latest do not support file_picker ^8.0.0+1,

**Comment by vdusautois-zilo** [comment]
Same issue here, any update or workaround anyone ?

**Comment by nirmalyohannan** [comment]
Yes, File Picker Dependency issue is occurring while using this package.
This won't be solved until author of the package increments the version of filePicker package used in pubspec.yaml.
![image](https://github.com/user-attachments/assets/f91e46d6-b074-4ae7-865a-34d73a6f0aa7)
Here it is still using FilePicker: 6.1.1

A workaround is to fork the repository and update versions yourself

**Comment by nirmalyohannan** [comment]
I have updated dependencies and created a pull request #215 . Don't know if this will be merged. 
Meanwhile you may use the package from [this repository](https://github.com/nirmalyohannan/quill_html_editor) if you need dependency updated package.
